src/lib/websocket/index.ts

```diff
@@ -0,0 +1,293 @@
+import WebSocket, { Server, Data } from 'ws';
+import * as http from 'http';
```

Comment:
```suggestion
import http from 'http';
```

This can be default-imported thanks to the `allowSyntheticDefaultImports` TS option that is active.